tests/integration/api/test_decision_environment.py

```diff
@@ -245,6 +253,66 @@ def test_partial_update_decision_environment(
         assert status_message in response.data["eda_credential_id"]
 
 
+@pytest.mark.parametrize(
+    ("inputs", "status_code", "status_message"),
+    [
+        (
+            {"username": "dummy-user", "password": "secret"},
+            status.HTTP_200_OK,
+            None,
+        ),
+        (
+            {
+                "username": "dummy-user",
+                "password": "secret",
+                "host": "quay.io",
+            },
+            status.HTTP_200_OK,
+            None,
+        ),
+        (
+            {
+                "username": "dummy-user",
+                "password": "secret",
+                "host": "registry.com",
+            },
+            status.HTTP_400_BAD_REQUEST,
+            "does not match with the credential host",
+        ),
+    ],
+)
+@pytest.mark.django_db
+def test_partial_update_decision_environment_with_image_url_and_host(
```

Comment:
Should we have a test as well when creating a DE?

Reply:
Added already at https://github.com/ansible/eda-server/pull/964/files#diff-66e5a86ad1ff6de7358f68e2e87359606462164415f5456570e127256188e313R110